src/commands/deployWorkspaceProject/internal/startingConfiguration/TryUseExistingRegistryStep.ts

```diff
@@ -37,6 +39,12 @@ export class TryUseExistingRegistryStep extends AzureWizardExecuteStep<DeployWor
             }
         }
 
+        if (registryInSameResourceGroup) {
+            ext.outputChannel.appendLog(localize('resourceGroupRegistry', 'Found an available registry "{0}" in the provided resource group "{1}".', registryInSameResourceGroup.name, context.resourceGroup?.name));
```

Comment:
Should these things being output also be displayed in the activity description? I feel like these progress messages are kind of useful (and it'd probably be worth to just make a helper function that outputs in the channel _and_ the activity notification)

My only concern is that it would flash by too quickly for users to see.

Reply:
So we do eventually show which ACR was used or created.  I kind of think that's good enough for the activity log.  I was thinking activity log is more high level, and output log is more detailed.  I don't think the activity log needs to tell the user how we got the resource, just which resource we are using or creating?